src/api/spaces/object.rs

```diff
@@ -0,0 +1,64 @@
+use http::header::HeaderValue;
+
+lazy_static! {
+    /// ACL header value for read-only public access.
+    pub static ref ACL_PUBLIC: HeaderValue = HeaderValue::from_static("public-read");
+    /// ACL header value for private access.
+    pub static ref ACL_PRIVATE: HeaderValue = HeaderValue::from_static("private");
+    /// Content-Disposition header value for displaying objects inline.
+    pub static ref DISPLAY_INLINE: HeaderValue = HeaderValue::from_static("inline");
+    /// Content-Disposition header value for displaying objects as attachments.
+    pub static ref DISPLAY_ATTACHMENT: HeaderValue = HeaderValue::from_static("attachment");
+}
+
+/// Access rule for an object.
+#[derive(Clone, Copy)]
+pub enum ObjectACL {
+    PublicRead,
+    Private,
+}
+
+/// Flag to indicate how the content should be displayed for an object.
+#[derive(Clone, Copy)]
+pub enum ContentDisposition {
+    Attachment,
+    Inline,
+}
+
+/* Helper impls */
+
+impl ObjectACL {
+    #[inline]
+    pub(crate) fn header(&self) -> HeaderValue {
+        match *self {
+            ObjectACL::PublicRead => ACL_PUBLIC.clone(),
```

Comment:
Is cloning required here? Could we pass a `&'static HeaderValue` back and let the caller choose to clone it?

Reply:
Yep, makes sense.